src/js/archive.js

```diff
@@ -18,7 +18,7 @@ function populateArchive() {
 
     // var repoName = (variant + '-releases');
 
-    loadJSON('openjdk-releases', 'releases', function(response) {
+    loadJSON('openjdk8-releases', 'releases', function(response) {
```

Comment:
OK we'll have to make this work for OpenJDK 9 as well at some point, but that's a later fix.